CrewChiefV4/carClassData.json

```diff
@@ -546,7 +546,7 @@
 		{
 			"carClassEnum": "TC1",
 			"raceroomClassIds": [4517, 6036, 6309],
-			"rf2ClassNames": ["Lada Vesta", "Honda Civic WTCC-TC1", "Chevrolet Cruze TC1", "Citroen C-Elysee", "Volvo S60 TC1"],
+			"rf2ClassNames": ["Lada Vesta", "Honda Civic WTCC-TC1", "Chevrolet Cruze TC1", "Citroen C-Elysee", "Volvo S60 TC1", "BMW 125i", "Ford Focus", "Chevrolet Cruze", "Toyota Avensis", "Civic BTCC 2013"],
```

Comment:
these new cars aren't TC1 (WTCC spec), they're TC2 (pre-2014 WTCC, TCR, BTCC etc)

Reply:
merged anyway, and i've moved these cars to TC2 in master